package.json

```diff
@@ -132,7 +132,7 @@
         "./node_modules/bootstrap-slider/dist/css/bootstrap-slider.css",
         "./node_modules/simplemde/dist/simplemde.min.css",
         "./node_modules/select2/dist/css/select2.min.css",
-        "./node_modules/select2-bootstrap4-theme/dist/select2-bootstrap4.scss",
+        "./node_modules/select2-bootstrap4-theme/src/select2-bootstrap4.scss",
```

Comment:
This change doesn't seem logical to me, has this been changed on purpose?

Reply:
Yeah, ~/dist/select2-boostrap4.scss doesn't excist which is why the styling of the select2 input fields is kind of weird. But I'll replace it with the .min.css just like the other libraries.